tests/osgi-tests/src/test/java/org/wso2/carbon/osgi/secvault/SecVaultOSGITest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.carbon.osgi.secvault;
+
+import org.ops4j.pax.exam.ExamFactory;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.ops4j.pax.exam.testng.listener.PaxExam;
+import org.testng.Assert;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+import org.wso2.carbon.container.CarbonContainerFactory;
+import org.wso2.carbon.secvault.SecureVault;
+import org.wso2.carbon.secvault.exception.SecureVaultException;
+
+import javax.inject.Inject;
+
+@Listeners(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+@ExamFactory(CarbonContainerFactory.class)
+public class SecVaultOSGITest {
+
+    @Inject
+    SecureVault secureVaultService;
+
+    String toBeEncrypted = "testPassword";
```

Comment:
Since "String toBeEncrypted = "testPassword" " is used only in one method we can move this variable inside the test method. WDYT?